src/components/opt-in-out/helpers/make-beta-welcome-toast.tsx

```diff
@@ -1,15 +0,0 @@
-import { toast, ToastColor } from 'components/toast'
-import { OptInPlatformOption, PlatformOptionTitles } from '../types'
-
-export function makeBetaWelcomeToast(optInPlatform: OptInPlatformOption) {
-	// Get the product name
-	const platformName = PlatformOptionTitles[optInPlatform]
-	// Make toast
-	toast({
-		color: ToastColor.highlight,
-		title: `Welcome to the ${platformName} Website Beta!`,
-		description:
-			'You can leave the Beta at anytime by clicking the Leave Beta button near the start of the main content area.',
```

Comment:
🤔 are they really sure they want to remove this toast while we still have beta opt in flows in effect? Seems odd to remove.

Reply:
I won't fight it but it just seems weird to me lol

Reply:
🤷‍♀️